src/Stryker.Core/Stryker.Core/Reporters/DashboardReporter.cs

```diff
@@ -32,27 +34,40 @@ public class DashboardReporter : IReporter
 
         public void OnAllMutantsTested(IReadOnlyProjectComponent reportComponent, TestProjectsInfo testProjectsInfo)
         {
+            if (_options.ReportTypeToOpen == ReportType.Dashboard)
+            {
+                _dashboardClient.PublishFinished().Wait();
```

Comment:
We should probably only do this after we've actually uploaded the completed report.